src/lib/alarmMgr/AlarmManager.cpp

```diff
@@ -0,0 +1,240 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <string>
+#include <map>
+
+#include "logMsg/logMsg.h"
+#include "logMsg/traceLevels.h"
+
+#include "alarmMgr/AlarmManager.h"
+
+
+
+/* ****************************************************************************
+*
+* AlarmManager::AlarmManager - 
+*/
+AlarmManager::AlarmManager():
+dbOk(true),
```

Comment:
I did indent like you say to start with.
It looks really funny:

    AlarmManager::AlarmManager():
      dbOk(true),
      xxx(0)
    {
      ....
    }

In my opinion it looks better like this.
Alternative: indent until the ':' of "AlarmManager::AlarmManager():"

This is a decision for the style-guide.
Let's make a decision ...